src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt

```diff
@@ -38,6 +42,20 @@ class AdmStormStartupActivity : ProjectActivity {
             return
         }
 
+        //TODO: remove it after release AdmStorm on Windows
+        if (SystemInfo.isWindows) {
```

Comment:
Let's put it in a separate file, as we did for `ChangeSshBackendStartup`, `PluginsUpdateStartup`. It will not overload an already overloaded class.

Reply:
Ok. Fixed